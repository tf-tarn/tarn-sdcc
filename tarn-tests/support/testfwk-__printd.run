./sim/sim -q -h -s 10000000 $program_file > $output