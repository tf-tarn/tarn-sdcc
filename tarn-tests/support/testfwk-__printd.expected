OUTPUT=30
OUTPUT=39
OUTPUT=39
OUTPUT=31
OUTPUT=31
OUTPUT=32
OUTPUT=33
OUTPUT=34
OUTPUT=35
OUTPUT=36
OUTPUT=37
OUTPUT=38
OUTPUT=39
OUTPUT=30
OUTPUT=31
OUTPUT=32
OUTPUT=2d
OUTPUT=2d
OUTPUT=2d
OUTPUT=20
OUTPUT=4f
OUTPUT=4b
OUTPUT=3a
OUTPUT=20
OUTPUT=22
OUTPUT=63
OUTPUT=6f
OUTPUT=6e
OUTPUT=64
OUTPUT=69
OUTPUT=74
OUTPUT=69
OUTPUT=6f
OUTPUT=6e
OUTPUT=20
OUTPUT=61
OUTPUT=74
OUTPUT=20
OUTPUT=3c
OUTPUT=66
OUTPUT=69
OUTPUT=6c
OUTPUT=65
OUTPUT=6e
OUTPUT=61
OUTPUT=6d
OUTPUT=65
OUTPUT=3e
OUTPUT=3a
OUTPUT=39
OUTPUT=39
OUTPUT=0a
